README.md

```diff
@@ -31,10 +31,36 @@
 * The API itself is really simple. You're task is to peek at the endpoints found in the `server.js` file and write docs for each one, then write the tests for the end points.
 
 ## DOCUMENTATION GOES HERE
+# User API
+This API holds access to a database of NES Games. It allows a user to sign in, add a new game, update an existing game, display a list of all games currently stored and delete games from the list.
+
+# To Start The Database
+  * If you haven't already, install MongoDB, then initialize MongoDB.
+  * Then, start the API by navigating to the root directory, running npm install to grab all dependencies, and then npm start to start the server.
+
+# API Endpoints:
+The server is accessible on:
+``` http://localhost:5050 ```
+# [POST]
```

Comment:
I would also add example snippets of the expected inputs/outputs to each endpoint in your documentation.